src/meta/api/src/schema_api_test_suite.rs

```diff
@@ -2484,17 +2492,51 @@ impl SchemaApiTestSuite {
             true,
         )
         .await?;
-        self.create_out_of_retention_time_table(
-            mt,
-            tbl_name_ident.clone(),
-            DBIdTableName {
-                db_id: res.db_id,
-                table_name: tb1_name.to_string(),
-            },
-            Some(Utc::now() - Duration::days(2)),
-            false,
-        )
-        .await?;
+        let table_id = self
+            .create_out_of_retention_time_table(
+                mt,
+                tbl_name_ident.clone(),
+                DBIdTableName {
+                    db_id: res.db_id,
+                    table_name: tb1_name.to_string(),
+                },
+                Some(Utc::now() - Duration::days(2)),
+                false,
+            )
+            .await?;
+
+        info!("--- create and get stage file info");
+        {
+            let stage_info = TableStageFileInfo {
+                etag: Some("etag".to_owned()),
+                content_length: 1024,
+                last_modified: Utc::now(),
+            };
+            let mut file_info = BTreeMap::new();
+            file_info.insert("file".to_string(), stage_info.clone());
+
+            let req = UpsertTableStageFileReq {
+                table: TableNameIdent {
+                    tenant: tenant1.to_string(),
+                    db_name: db1_name.to_string(),
+                    table_name: tb1_name.to_string(),
+                },
+                file_info: file_info.clone(),
+                expire_at: Some((Utc::now().timestamp() + 86400) as u64),
```

Comment:
Can we remove the expire_at?
I am not sure why snowflake has this limit?
For example:
A has a public dataset on s3, and B wants to load the increment files from A's bucket by `COPY into t1 FROM s3://B/bucket/`. When the expire reached after 86400, B will reload all the data, this is a big problem.

Reply:
> Can we remove the expire_at? I am not sure why snowflake has this limit? For example: A has a public dataset on s3, and B wants to load the increment files from A's bucket by `COPY into t1 FROM s3://B/bucket/`. When the expire reached after 86400, B will reload all the data, this is a big problem.

`UpsertTableStageFileReq.expire_at` is an option field, which can be set by the `schema_api` user, if copy into table stage file meta needs not to be expired, this field can set to be none.

Reply:
OK, let's set it to `None` make it never expired.
Thanks.